.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove the vscode. Why do we still have this I had talked about it in the last PR